v0.0.1/admin-schema.json

```diff
@@ -0,0 +1,633 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
What is the "$schema" property? And is this the right value for it?

Reply:
That is the [version of json schema](https://json-schema.org/specification.html) we are using. And indeed, while a valid version, we are using the latest version `2020-12`, I've updated in the documents that had the earlier version